docs/environment.yml

```diff
@@ -0,0 +1,12 @@
+# This environment is used by the RTD config for PR builds. RTD uses this as the
+# base environment and then adds in the sphinx etc tools on top.
+# See: https://docs.readthedocs.io/en/stable/guides/conda.html
+name: rtd
+channels:
+  - numba/label/dev
+dependencies:
```

Comment:
The [developer docs](https://numba.readthedocs.io/en/stable/developer/contributing.html#getting-set-up) also recommend installing jinja2 and cffi (and scipy as well, but I know there's no dependency on that in Numba) - is there any chance not having these present might result in some part of the docs getting omitted?

Reply:
Good question. I don't think so on the basis that the `requirements.txt` which the docs were building against, prior to them being broken by lack of wheels, do not contain these.
https://github.com/numba/numba/blob/92861d01a5f331f4262d5c7d1798b3f7517086b7/requirements.txt
https://github.com/numba/numba/blob/92861d01a5f331f4262d5c7d1798b3f7517086b7/docs/requirements.txt

`argparse` is in `<numba root>/requirements.txt` but not mentioned in the docs.

Reply:
I think `argparse` was only needed for Python < 3.3 (or thereabouts), and it's now part of the standard library.

Reply:
ah yes, thanks. https://docs.python.org/3/library/argparse.html 
> _New in version 3.2_

 I think https://github.com/numba/numba/pull/6508 sorts this out anyway by just deferring to `setup.py`.